homeassistant/components/media_player/services.yaml

```diff
@@ -165,6 +165,23 @@ play_media:
       selector:
         boolean:
 
+browse_media:
+  target:
+    entity:
+      domain: media_player
+  fields:
+    media_content_type:
+      required: false
+      example: "music"
+      selector:
+        text:
```

Comment:
As a double check, is this standardized?

Reply:
The most command definition is

```
    media_content_type: MediaType | str | None = None
```

Integration sometimes use the constants from MediaType or their own strings or a mix of both.

We could populate the list with the constants?

Reply:
I am not sure if this is possible, but it would be a good start I think

Reply:
From what I can see - If we provide a list; then the user can only select from the list. There is no way to allow them to select from the list or enter their own string.  Since there is no single standardized list that all integrations support, we should leave it as a free form text field and provide examples in the docs (which I will work on next).